include/kitty/operations.hpp

```diff
@@ -125,6 +125,17 @@ inline bool equal( const TT& first, const TT& second )
   return binary_predicate( first, second, std::equal_to<>() );
 }
 
+/*! \brief Checks if first truth table implies a second truth table
+
+  \param first First truth table
+  \param second Second truth table
+*/
+template<typename TT>
+inline bool implies( const TT& first, const TT& second )
+{
+  return is_const0( ~(( ~first ) | second ) );
```

Comment:
`operations.hpp` does not include `operators.hpp`. It would be better to implement this operation based on the iterators.

Reply:
Agreed. However, I do not have a particular strong interest in improving this implementation. The `implies` function could also remain as ann implementation detail in `mockturtle`.